### ISSUE TITLE
[FEATURE] Add ability: Automatically open njump links in Amethyst

### ISSUE THREAD

**Issue by summoner001** [vitorpamplona/amethyst#1217]
**Describe the solution you'd like**

When tap on an njump-link in a browser or a messenger (simplex chat) automatically open the link in Amethyst.

**Comment by vitorpamplona** [comment]
I am not sure that is possible anymore. Deep links like that require us to prove that we control the domain name. :(

**Comment by davotoula** [comment]
> **Describe the solution you'd like**
> 
> When tap on an njump-link in a browser or a messenger (simplex chat) automatically open the link in Amethyst.

This might be possible by adding njump.me as supported links to amethyst and parsing the link in amethyst and opening profile / note etc.

It will not be a verified link but it can be done.

NewPipe does this with YouTube.com links.

But do we want to tie amethyst to a single external website like njump?

**Comment by KotlinGeekDev** [comment]
@davotoula  I see what you mean.


> I am not sure that is possible anymore. Deep links like that require us to prove that we control the domain name. :(


Another idea might be to add support to njump instead, and have a crafted URI of the form 'amethyst://<npub, nprofile, etc>' and use that to open into Amethyst.

**Comment by vitorpamplona** [comment]
I think they already do that. But it requires loading the page and then opening to the default nostr app

**Comment by davotoula** [comment]
Yes they do that already from njump website, it works well on mobile browser.

I guess this feature request is to open njump links from other sites.

**Comment by vitorpamplona** [comment]
Yeah, the goal would be to short cut it. But I am not sure if it is possible. 

We can talk to njump domain owners and manually get them to verify each app

**Comment by summoner001** [comment]
Yes, there is a button on the Njump page to open it in Amethyst. But if I get a njump link in a messenger like SimpleX Chat. It opens the njump link automatically in amethyst and does not load the web page.

For example, if I get a youtube link in SimpleX, the link opens automatically in Tubular/newpipe.

**Comment by KotlinGeekDev** [comment]
> I think they already do that. But it requires loading the page and then opening to the default nostr app

I see. So @davotoula 's idea seems better(?).

**Comment by kirpy0x** [comment]
I don't think opening from a browser is possible. I've never experienced deeplinking to affect browsers unless the website you're on specifically prefers it.

I send a PR to make opening from other apps. Tested from Nextcloud Talk.
